snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/datastore/request/SnomedConceptSearchRequestBuilder.java

```diff
@@ -106,17 +106,6 @@ public final SnomedConceptSearchRequestBuilder filterBySemanticTags(Iterable<Str
 		return addOption(SnomedConceptSearchRequest.OptionKey.SEMANTIC_TAG, semanticTags);
 	}
 
-	/**
-	 * Filter matches by the specified Expression Constraint Language (ECL) expression. The currently supported ECL version is v1.1. See
-	 * <a href="http://snomed.org/ecl">ECL Specification and Guide</a>.
-	 * 
-	 * @param expression - the ECL expression
-	 * @return SnomedConceptSearchRequestBuilder
-	 */
-	public final SnomedConceptSearchRequestBuilder filterByEcl(String expression) {
-		return addOption(SnomedConceptSearchRequest.OptionKey.ECL, expression);
-	}
-
 	/**
 	 * Filter matches in the stated tree by the specified Expression Constraint Language (ECL) expression. The currently supported ECL version is
 	 * v1.3. See <a href="http://snomed.org/ecl">ECL Specification and Guide</a>.
```

Comment:
The reference to the ECL specification version should be updated or removed entirely.

Reply:
Removed it, since it is heavily depends on the version of the termserver.